ocaml/xapi/helpers.ml

```diff
@@ -1287,3 +1287,44 @@ let retry ~__context ~doc ?(policy = Policy.standard) f =
 
 let retry_with_global_lock ~__context ~doc ?policy f =
   retry ~__context ~doc ?policy (fun () -> with_global_lock f)
+
+let vswitch_protocol_to_string = function
```

Comment:
Is this code using a different indentation scheme than usual? Could you run this trough ocp-indent?